### PR TITLE
AjitLeadCommit

### PR DESCRIPTION
**Pull request by shrikantkatkur**

### CODE REVIEW

File: src/test/java/com/test/ShrikantDemoTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import org.testng.annotations.Test;
+
+@Test
+public class ShrikantDemoTest {
+	@Test
+	public void test1() {
+		WebDriver driver = new FirefoxDriver();
+		driver.get("https://www.myntra.com/");
+		String title = driver.getTitle();
+		boolean result = title.contains("Myntra");
```
Comment (Ajiket97) [inline src/test/java/com/test/ShrikantDemoTest.java]: please check once again.

**Review by Ajiket97 — COMMENTED** [review]
hh

File: src/test/java/com/test/ShrikantDemoTest.java (hunk 1 above)
Comment (shrikantkatkur) [reply src/test/java/com/test/ShrikantDemoTest.java]: please specify

File: src/test/java/com/test/ShrikantDemoTest.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test
+public class ShrikantDemoTest {
+	@Test
+	public void test1() {
+		WebDriver driver = new FirefoxDriver();
+		driver.get("https://www.myntra.com/");
+		String title = driver.getTitle();
```
Comment (Ajiket97) [inline src/test/java/com/test/ShrikantDemoTest.java]: title veriable is alredy use  pleas check agains

**Review by Ajiket97 — APPROVED** [review]

**Pull request merged**